test/specs/elements/Header/_Header-test.js

```diff
@@ -1,13 +1,17 @@
+import React from 'react'
+
 import _Header from 'src/elements/Header/_Header'
 import * as common from 'test/specs/commonTests'
 
 describe('_Header', () => {
-  common.propKeyOnlyToClassName(_Header, 'left', { className: 'aligned' })
-  common.propKeyOnlyToClassName(_Header, 'center', { className: 'aligned' })
-  common.propKeyOnlyToClassName(_Header, 'right', { className: 'aligned' })
-  common.propKeyOnlyToClassName(_Header, 'justified')
+  common.propKeyAndValueToClassName(_Header, 'aligned')
   common.propKeyOnlyToClassName(_Header, 'dividing')
   common.propKeyOnlyToClassName(_Header, 'block')
   common.propKeyOnlyToClassName(_Header, 'attached')
   common.propKeyOnlyToClassName(_Header, 'floating')
+
+  it('does not have aligned class when justified', () => {
+    shallow(<_Header aligned='justified' />)
+      .should.not.have.className('aligned')
+  })
```

Comment:
Once updated and the new prop util and common test are used, you can delete this test:

```diff
-
-  it('does not have aligned class when justified', () => {
-    shallow(<_Header aligned='justified' />)
-      .should.not.have.className('aligned')
-  })
```